server/worker_pool.go

```diff
@@ -99,6 +108,9 @@ func (wp *workerPool) Destroy() {
 	// Destroy the pools
 	wp.Other.Destroy()
 	wp.ECDSA.Destroy()
+	wp.RSA.Destroy()
```

Comment:
hmm, rsa pool wasn't destroyed before.

Reply:
nothing bad happened from it AFAIK,  but who knows?